pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -156,6 +156,7 @@ def close(self) -> None:
 
     def parse_test_menu(
         self,
+        strip: bool = True,
```

Comment:
when adding an extra argument, it's better to add it at the very last with a default value to avoid breaking changes.

and please also improve the naming here. `strip` is more like a word for removing white spaces. how about `include_ignored` with a default value `False`?